Fix a minor type error in the perminit calls in the Triple DES implementation.

https://sourceforge.net/p/mcrypt/patches/15/
--- modules/algorithms/tripledes.c.orig
+++ modules/algorithms/tripledes.c
@@ -201,8 +201,8 @@ static int _mcrypt_desinit(TRIPLEDES_KEY * key)
 	spinit(key, 0);
 	spinit(key, 1);
 	spinit(key, 2);
-	perminit(&key->iperm, ip);
-	perminit(&key->fperm, fp);
+	perminit(key->iperm, ip);
+	perminit(key->fperm, fp);
 
 
 	return 0;
